app/javascript/components/App.js

```diff
@@ -6,7 +6,7 @@ const App = () => {
   return (
     <Router>
       <Switch>
-        <Route exact path="/" component={Greeting} />
+        <Route exact path="/" component="<Greeting/>" />
```

Comment:
- [ ] Kindly make sure the greeting message is rendered as requested by the previous reviewer. Currently your react componentes are not apprearing in the browser, and your redux store is not connected to your components .